Hearthstone Deck Tracker/Config.cs

```diff
@@ -151,7 +151,7 @@ public class Config
 		[DefaultValue(true)]
 		public bool ConstructedAutoImportNew = true;
 
-		[DefaultValue(true)]
+		[DefaultValue(true)]

```

Comment:
yea, inconsistent line ending. That specific line uses LF line ending, while all other lines end with CRLF. Should I revert this change or leave it (for consistency?).